addons/cookoff/functions/fnc_cookOff.sqf

```diff
@@ -113,11 +113,20 @@ if (local _vehicle) then {
 
         [_vehicle, _fnc_FlameEffect, 12] call _fnc_FlameEffect; // recursive function
 
+        private _randomPosition = _vehicle getPos [100, random 360];
+
         {
-            if (local _x && {!(_x call EFUNC(common,isPlayer))}) then {
-                _x action ["Eject", _vehicle];
+            _x leaveVehicle _vehicle;
+            
+            if (leader _x == _x) then {
+                private _wp = group _x addWaypoint [_randomPosition, 0, 1];
```

Comment:
`currentWaypount _group` Will make it their active waypoint. Also it was me who proposed that @commy2, but I'm quite happy to be be mistaken for pabst :laughing:

Oh and index 0 isn't _exactly_ reserved for anything, there's some weird behaviour regarding it, but you can definitely manipulate the waypoint at index 0 via scripting. The reason it wouldn't work as the current waypoint is because completed waypoints also take up indices - nothing decrements unless one is deleted.